### PR TITLE
Update ca.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**

### CODE REVIEW

**Review by dct-infra — APPROVED** [review]

**Comment by taodicakhia** [comment]
already in #4780

**Review by freearhey — APPROVED** [review]

File: channels/ca.m3u [hunk 1]
```diff
@@ -205,7 +205,7 @@ http://live.tamilvision.tv:8081/TVI/SD/playlist.m3u8
 #EXTINF:-1 tvg-id="TeleCulturelleMedias.ca" tvg-country="CA" tvg-language="French" tvg-logo="https://i.imgur.com/oRigj9q.jpg" group-title="",Télé Culturelle Médias (720p)
 https://5790d294af2dc.streamlock.net/8150/8150/playlist.m3u8
 #EXTINF:-1 tvg-id="TeleMag.ca" tvg-country="CA" tvg-language="French" tvg-logo="https://i.imgur.com/i5Isrob.png" group-title="General",Télé-Mag Québec (1080p) [Not 24/7]
-https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/user/ctmq/live
+https://iptv.dc-team.com/query?url=https://www.youtube.com/user/ctmq/live
```
Comment (jkemp814) [inline channels/ca.m3u:208]: Whoa there ! I know you like that service, but there's no need to spam me ! Let the server breathe a little bit (RATE LIMIT EXCEEDED)

File: channels/ca.m3u [hunk 2]
```diff
@@ -217,7 +217,7 @@ https://tscstreaming-lh.akamaihd.net/i/TSCLiveStreaming_1@91031/master.m3u8
 #EXTINF:-1 tvg-id="TV16Toronto.ca" tvg-country="CA" tvg-language="English" tvg-logo="https://i.imgur.com/brxkNbw.png" group-title="Local",TV 16 Toronto (720p) [Not 24/7]
 http://rtmp.smartstream.video:1935/capco/tv29/playlist.m3u8
 #EXTINF:-1 tvg-id="TVC9.ca" tvg-country="CA" tvg-language="French" tvg-logo="https://i.imgur.com/KLWdogU.png" group-title="Local",TVC9 (1080p) [Not 24/7]
-https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UC0JP0ek-HhcqisiEpsZiQZA/live
+https://iptv.dc-team.com/query?url=https://www.youtube.com/channel/UC0JP0ek-HhcqisiEpsZiQZA/live
```
Comment (jkemp814) [inline channels/ca.m3u:220]: Whoa there ! I know you like that service, but there's no need to spam me ! Let the server breathe a little bit (RATE LIMIT EXCEEDED)

**Review by jkemp814 — APPROVED** [review]
iptv.dc-team.com/ finally caught its breath stream forwarded on.

**Pull request merged**